src/library/driftory.ts

```diff
@@ -164,7 +166,8 @@ export default class Driftory {
     }
   }
 
-  openComic(comic: Comic) {
+  openComic(unsafeComic: Comic | string) {
+    const { comic } = typeof unsafeComic === "string" ? JSON.parse(unsafeComic) as Comic : unsafeComic
```

Comment:
Allows people to pass in the comic JSON directly as a string without having to parse it first.